public/src/js/widgets/columns/fronts-config.html

```diff
@@ -259,10 +259,12 @@
                 <span class="cnf-form__value" data-bind="text: meta.groups"></span>
             <!-- /ko -->
 
-			<!-- ko if: meta.type() === "flexible/general"-->
-			<label for="flexGenStoryLimit">Max stories to display</label>
-			<input id="flexGenStoryLimit" type="number" data-bind="value: meta.displayHints.maxItemsToDisplay" attr: { max: 20, min: 1 } max="20" min="1" placeholder="9">
-			<!-- /ko -->
+            <!-- ko if: meta.type() === "flexible/general"-->
+              <div data-bind="foreach: meta.groupsConfig">
+                <label data-bind="attr: { for: name }"><span data-bind="text: name" class="capitalize-first-letter"></span> stories</label>
+                <input type="number" data-bind="value: maxItems, attr: { id: name, max: 20, min: 0 }" max="20" min="0">
```

Comment:
I don't think `id`s can have whitespace. Perhaps this will work (untested):
`attr: { for: 'maxItems-' + $index }`